src/test/java/org/shikshalokam/backend/scp/SelfCreationPortalCreateNewPermission.java

```diff
@@ -48,7 +47,7 @@ public void testCreateNewPermission() {
 
         // Updated request body
         String requestBody = "{ " +
-                "\"code\": \"project_subash_uat\", " +
+                "\"code\": \"project_test_break\", " +
```

Comment:
Please use code like this to create a Json object and pass the same as body to the request 

JSONObject jsonObject = new JSONObject();

        // Add key-value pairs
        jsonObject.put("name", "John Doe");
        jsonObject.put("age", 30);
        jsonObject.put("email", "johndoe@example.com");
        jsonObject.put("isMarried", false);